cosmoz-theme-switcher.js

```diff
@@ -1,8 +1,7 @@
-/*global Polymer, document */
 (function () {
-
 	'use strict';
 
+	/*eslint no-unused-vars: ["warn", { "argsIgnorePattern": "event" }]*/
```

Comment:
Interesting! We should probably have this as a part of the global ruleset. I like to leave unused event arguments to highlight that it's and event handler.